@ornikar/eslint-config-react/rules/react-testing-library-base.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  plugins: ['testing-library'],
+  rules: {
+    // Allows consistent data-testid and testID
+    // Format is: sectionName.pageName?.FileName.uniqueIdentifier
+    'testing-library/consistent-data-testid': [
+      'error',
+      {
+        testIdPattern: '^([a-z][a-zA-Z]+\\.)+{fileName}\\.([a-z][a-zA-Z]+)+$',
+        testIdAttribute: ['data-testid', 'testID'],
```

Comment:
Plutôt `testId` pour le attribute ?
```suggestion
        testIdAttribute: ['data-testid', 'testId'],
```

Reply:
c'est pas le nom d'un champ random, mais celui de react-native : https://reactnative.dev/docs/view#testid